Fix:
ip-lib.c:55:36: error: implicit declaration of function 'inet_ntoa' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
--- ip-lib.c.orig	2005-12-12 02:37:19.000000000 -0600
+++ ip-lib.c	2021-07-27 09:36:29.000000000 -0500
@@ -21,6 +21,7 @@
 
 */
 
+#include <arpa/inet.h>
 #include <stdlib.h>
 #include <stdio.h>
 #include <string.h>
